relay.go

```diff
@@ -544,12 +549,12 @@ func (r *Relayer) handleNonCallReq(f *Frame) error {
 	// Stop the timeout if the call if finished.
 	item, stopped, ok := items.Get(f.Header.ID, finished /* stopTimeout */)
 	if !ok {
-		return errUnknownID
+		return _frameNotDropped, errUnknownID
 	}
 	if item.tomb || (finished && !stopped) {
 		// Item has previously timed out, or is in the process of timing out.
 		// TODO: metrics for late-arriving frames.
-		return nil
+		return !finished, nil
```

Comment:
similar to above, if the frame isn't being forwarded, so it is being dropped right?